pom.xml

```
@@ -17,7 +17,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <failOnMissingWebXml>false</failOnMissingWebXml>
 
-        <vaadin.version>15.0.5</vaadin.version>
+        <vaadin.version>16.0-SNAPSHOT</vaadin.version>
```

I am not sure if you should use `16.0.0.beta1` here or leave it as snapshot.
```suggestion
        <vaadin.version>16.0.0.beta1</vaadin.version>
```

---

Bumped now.